tests/e2e/support/objects/app-admin-settings/spaces/actions.ts

```diff
@@ -49,39 +52,68 @@ export const changeSpaceQuota = async (args: {
     page.locator(actionConfirmButton).click()
   ])
 }
+export const clearSelection = async (args: { page: Page }): Promise<void> => {
+  const { page } = args
+  try {
+    await page.locator(clearSelectionBtn).click({ timeout: 500 })
+  } catch (ex) {}
+}
```

Comment:
Why wrap this in a try-catch?

Reply:
Because the clear selection is not always there before making a new selection